### ISSUE TITLE
dependsOn clears input

### ISSUE THREAD

**Issue by tn34de** [madeyourday/contao-rocksolid-custom-elements#174]
I have used the dependsOn option successfully before. However, on a current project, it keeps clearing the input whenever the configuration contains multiple dependOn conditions. I cannot find any reason for this behaviour, as it has been working with a similar config before.

Contao 5.3.11
Custom Elements 2.4.10

```php
<?php

include_once \App\Helper\CustomHelper::getRootPath().'/templates/custom/rsce-config/special-headline.php';

return array(
    'label' => array('Hero', ''), 
    'types' => array('content'),
    'contentCategory' => 'Custom Elements',
    'standardFields' => array('cssID'),
    'fields' => array_merge(

        array (
            'heroStyle' => array(
                'label' => array('Hero-Style', ''),
                'inputType' => 'radio',
                'options' => array(
                    'standard' => 'Standard', 
                    'special' => 'Special (z.B. Startseite)', 
                ),
            ),
        ),

        getSpecialHeadlineSettings(),

        array (
            'textBlock' => array(
                'label' => array('Text-Einstellungen', ''),
                'inputType' => 'group',
                //'eval' => array('collapsed' => true),
                'dependsOn' => array(
                    'field' => 'heroStyle',
                    'value' => 'special', 
                ),
            ),
            'text' => array(
                'label' => array('Text', ''),
                'eval' => array(
                    'rte' => 'tinyMCE',
                    'allowHtml' => 'true',
                ),
                'inputType' => 'textarea',
                'dependsOn' => array(
                    'field' => 'heroStyle',
                    'value' => 'special', 
                ),
            ),
            'hyphens' => array(
                'label' => array('Automatische Umbrüche deaktivieren', 'Wörter werden nicht mehr durch CSS umgebrochen wenn der Platz nicht ausreicht.'),
                'inputType' => 'checkbox',
                'dependsOn' => array(
                    'field' => 'heroStyle',
                    'value' => 'special', 
                ),
            ),
             
            'imgBlock' => array(
                'label' => array('Bild-Einstellungen', ''),
                'inputType' => 'group',
                'dependsOn' => array(
                    'field' => 'heroStyle',
                    'value' => 'special', 
                ),
            ),

            'mainImage' => array(
                'label' => array('Bild', ''),
                'inputType' => 'fileTree',
                'eval' => array(
                    'fieldType' => 'radio',
                    'filesOnly' => true,
                    'extensions' => 'jpg,jpeg,png,gif,svg',
                ),
                'dependsOn' => array(
                    'field' => 'heroStyle',
                    'value' => 'special', 
                ),
            ),
            'bgBlock' => array(
                'label' => array('Hintergrund-Einstellungen', ''),
                'inputType' => 'group',
                'dependsOn' => array(
                    'field' => 'heroStyle',
                    'value' => 'standard', 
                ),
            ),

            'bgImage' => array(
                'label' => array('Bild', ''),
                'inputType' => 'fileTree',
                'eval' => array(
                    'fieldType' => 'radio',
                    'filesOnly' => true,
                    'extensions' => 'jpg,jpeg,png,gif,svg',
                ),
                'dependsOn' => array(
                    'field' => 'heroStyle',
                    'value' => 'standard', 
                ),
            ),
        ),
    ), 
);
```

**Comment by ausi** [comment]
> …it keeps clearing the input whenever the configuration contains multiple dependOn conditions.

What does that mean? At which point does the input get cleared? Directly after saving?

**Comment by ausi** [comment]
Fixed in version 2.4.11

**Comment by tn34de** [comment]
> > …it keeps clearing the input whenever the configuration contains multiple dependOn conditions.
> 
> What does that mean? At which point does the input get cleared? Directly after saving?

Yes, directly at saving. Installed 2.4.11 - it's working now. Thanks!
